connectors/src/connectors/intercom/index.ts

```diff
@@ -756,3 +756,22 @@ export async function pauseIntercomConnector(connectorId: ModelId) {
 
   return new Ok(undefined);
 }
+
+export async function unpauseIntercomConnector(connectorId: ModelId) {
+  const connector = await ConnectorResource.fetchById(connectorId);
+  if (!connector) {
+    logger.error({ connectorId }, "[Intercom] Connector not found.");
+    return new Err(new Error("Connector not found"));
+  }
+
+  await connector.markAsUnpaused();
+
+  const r = await launchIntercomSyncWorkflow({
```

Comment:
wdyt @PopDaph  ?

Reply:
I would do this instead -> so that when we launch the workflow we signal that it needs to go through all convos (since we paused the conversations webhook). 

```
const teamsIds = await IntercomTeam.findAll({
    where: {
      connectorId,
    },
    attributes: ["teamId"],
  });
  const toBeSignaledHelpCenterIds = helpCentersIds.map((hc) => hc.helpCenterId);

const r = await launchIntercomSyncWorkflow({
    connectorId,
    teamIds: toBeSignaledTeamIds,
  });
```

Reply:
awesome thank you